doc/man/man8/daos.8

```diff
@@ -196,6 +199,17 @@ The \fBRESOURCE\fRs, respective \fBCOMMAND\fRs and \fBOPTION\fRs supported by \f
 .br
 	  \fB--epcrange=\fRB-E     container epoch range (destroy-snap)
 .TP
+.I filesystem \fR(\fIfs\fR) \fBCOMMAND\fRs:
+	  \fBcopy\fR           	 copy data to/from a container to POSIX path
+.TP
+.I filesystem \fR(\fIfs\fR) \fBOPTION\fRs:
+.br
+	  \fB--src=\fRSTR     src=daos://<pool/cont> | <path>
+.br
+	  \fB--dst=\fRSTR     dst=daos://<pool/cont> | <path>
```

Comment:
@dsikich Looking at this with fresh eyes...
Maybe we should clarify that a daos src/dst can have a path as well?
```suggestion
	  \fB--src=\fRSTR     src=daos://<pool/cont>[/<path>] | <path>
.br
	  \fB--dst=\fRSTR     dst=daos://<pool/cont>[/<path>] | <path>
```